clusters/resource_library_test.go

```diff
@@ -71,6 +72,13 @@ func TestLibraryDelete(t *testing.T) {
 					State: ClusterStateRunning,
 				},
 			},
+			{
+				Method:   "POST",
```

Comment:
Just double checking here: if the cluster is already running (as is returned in the last response), why do we try to start it again?

Reply:
Yeah good point let me change that